index.js

```diff
@@ -0,0 +1,122 @@
+})
+
+let refresh = document.querySelector("#refresh")
+const refreshPage=()=>{
+    window.location.reload();   
+}
+
+const generalJoke = ()=>{
+    fetch("https://cors-anywhere.herokuapp.com/https://official-joke-api.appspot.com/jokes/general/ten").then(res=>{
+        if(!res.ok){
+            throw Error(res.statusText="its Not Ok")
+        }
+        // debugger
+        return res.json()
+    }).then(res=>{
+        res.forEach(element => {
+            // debugger;
+            let li =document.createElement("li");
+            li.innerText=element.setup;
+            let jokeCard2= document.createElement("button");
+            jokeCard2.innerText="Joke Card";
+            jokeCard2.addEventListener("click", () => {
+                if(jokeCard2.innerText !== "Joke Card"){
+                    jokeCard2.innerText = "Joke Card"
+                } else{
+                jokeCard2.innerText = element.punchline
+                }
+            });
+            document.body.appendChild(li)
+            document.body.appendChild(jokeCard2)
+    
+        });
+    }).catch(err =>{
+        debugger
+    })
+    }
+
+const randomJoke =()=>{
+    fetch("https://cors-anywhere.herokuapp.com/https://official-joke-api.appspot.com/random_ten").then(res=>{
+        if(!res.ok){
+            throw Error(res.statusText="its Not Ok")
+        }
+        // debugger
+        return res.json()
+}).then(res=>{
+    res.forEach(element => {
+        // debugger;
+        let li =document.createElement("li");
+        li.innerText=element.setup;
+        let jokeCard2= document.createElement("button");
+        jokeCard2.innerText="Joke Card";
+        jokeCard2.addEventListener("click", () => {
+            if(jokeCard2.innerText !== "Joke Card"){
+                jokeCard2.innerText = "Joke Card"
+            } else{
+            jokeCard2.innerText = element.punchline
+            }
+        });
+        document.body.appendChild(li)
+        document.body.appendChild(jokeCard2)
+
+    });
+}).catch(err =>{
+    debugger
+})
+}
+
+const programJoke =()=>{
+    fetch("https://cors-anywhere.herokuapp.com/https://official-joke-api.appspot.com/jokes/programming/ten").then(res=>{
+        if(!res.ok){
+            throw Error(res.statusText="its Not Ok")
+        }
+        // debugger
+        return res.json()
+}).then(res=>{
+    res.forEach(element => {
+        // debugger;
+        let li =document.createElement("li");
+        li.innerText=element.setup;
+        let jokeCard2= document.createElement("button");
+        jokeCard2.innerText="Joke Card";
+        jokeCard2.addEventListener("click", () => {
+            if(jokeCard2.innerText !== "Joke Card"){
+                jokeCard2.innerText = "Joke Card"
+            } else{
+            jokeCard2.innerText = element.punchline
+            }
+        });
+        document.body.appendChild(li)
+        document.body.appendChild(jokeCard2)
+
+    });
+}).catch(err =>{
+    debugger
```

Comment:
Remove debuggers, having `console.log(err)` would be a good default way to know if an error occurs